src/app/clusters/identify-server/identify-server.h

```diff
@@ -0,0 +1,43 @@
+/**
+ *
+ *    Copyright (c) 2021 Project CHIP Authors
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+
+#pragma once
+
+#include <app-common/zap-generated/enums.h>
+#include <app/util/basic-types.h>
+
+struct Identify
+{
+    using onIdentifyStartCb    = void (*)(Identify *);
+    using onIdentifyStopCb     = onIdentifyStartCb;
+    using onEffectIdentifierCb = onIdentifyStartCb;
+
+    Identify(chip::EndpointId endpoint, onIdentifyStartCb onIdentifyStart, onIdentifyStopCb onIdentifyStop,
```

Comment:
So is the idea that the application will instantiate these objects?  That should probably be documented....

Reply:
done

Reply:
Not really done, @eve-cxrp.  The point is that something needs to document that the application is expected to create its own `Identify` object instances, with the right endpoint ids....

Reply:
Ok, you are right. That one I missed